src/benchmarks/micro/corefx/System.Console/Perf.Console.cs

```diff
@@ -13,17 +14,36 @@ namespace System.ConsoleTests
     /// - OpenStandardInput, OpenStandardOutput, OpenStandardError
     /// - ForegroundColor, BackgroundColor, ResetColor
     /// </summary>
-    [GcForce(true)] // forces full GC cleanup after every iteration, so streams allocated in OpenStandard* benchmarks are going to be finalized
     public class Perf_Console
     {
+        private readonly Consumer consumer = new Consumer();
+
         [Benchmark]
-        public Stream OpenStandardInput() => Console.OpenStandardInput();
+        public void OpenStandardInput()
+        {
+            using (var input = Console.OpenStandardInput())
+            {
+                consumer.Consume(input);
```

Comment:
I don't think consuming the variable is necessary here. the JIT can't know that OpenStandardInput and the destructor you're asking to be called don't have side effects, so they'll get called regardless. It's not a ton of extra work but if we don't need it why have it?

Reply:
I prefer to add it in defensive way to not rely on JIT implementation details (just always return the result to be consumed by the harness or consume it on your own)